### PR TITLE
Use Okio's Utf8.size.

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

File: wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java [hunk 1]
```diff
@@ -375,7 +375,7 @@ public String toString(E value) {
   public static final ProtoAdapter<String> STRING = new ProtoAdapter<String>(
       FieldEncoding.LENGTH_DELIMITED, String.class) {
     @Override public int encodedSize(String value) {
-      return utf8Length(value);
+      return (int) Utf8.size(value);
```
Comment (NightlyNexus) [inline wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java]: if this is even a change you want, should this assert that it's an int?

Comment (swankjesse) [inline wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java]: ooh, good question. Lemme add a task to test that.

Comment (swankjesse) [inline wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java]: https://github.com/square/wire/issues/670

**Comment by swankjesse** [comment]
Much better!

**Pull request merged**